app/components/molecules/promotion-card/promotion-card.styles.tsx

```diff
@@ -0,0 +1,51 @@
+import styled from 'styled-components';
+
+export interface PromotionCardProps {
+  variant?: 'default' | 'recommendation' | 'seasonal';
```

Comment:
I believe there's no need to make this `variant?: ...` since youre giving what it only be, you could remove it  and be just `variant: ...`